test/graphql/mutation/business-details.test.js

```diff
@@ -186,46 +186,226 @@ describe('business', () => {
+          line1: 'new line1',
+          line2: 'new line2',
+          line3: 'new line3',
+          line4: 'new line4',
+          line5: 'new line5',
+          pafOrganisationName: 'new pafOrganisationName',
+          postalCode: 'new postalCode',
+          street: 'new street',
+          uprn: 'new uprn'
+        }
+      },
+      isCorrespondenceAsBusinessAddress: true
+    }
+
+    const putPayloadOverrides = {
+      address: {
+        address1: 'new line1',
+        address2: 'new line2',
+        address3: 'new line3',
+        address4: 'new line4',
+        address5: 'new line5',
+        pafOrganisationName: 'new pafOrganisationName',
         buildingNumberRange: 'new buildingNumberRange',
+        buildingName: 'new buildingName',
+        flatName: 'new flatName',
+        street: 'new street',
         city: 'new city',
-        country: 'new country',
         county: 'new county',
+        postalCode: 'new postalCode',
+        country: 'new country',
+        uprn: 'new uprn',
         dependentLocality: 'new dependentLocality',
         doubleDependentLocality: 'new doubleDependentLocality',
-        flatName: 'new flatName',
-        line1: 'new line1',
-        line2: 'new line2',
-        line3: 'new line3',
-        line4: 'new line4',
-        line5: 'new line5',
-        pafOrganisationName: 'new pafOrganisationName',
-        postalCode: 'new postalCode',
-        street: 'new street',
-        uprn: 'new uprn'
+        addressTypeId: undefined
       },
       correspondenceAddress: {
-        buildingName: 'new buildingName',
+        address1: 'new line1',
+        address2: 'new line2',
+        address3: 'new line3',
+        address4: 'new line4',
+        address5: 'new line5',
+        pafOrganisationName: 'new pafOrganisationName',
         buildingNumberRange: 'new buildingNumberRange',
+        buildingName: 'new buildingName',
+        flatName: 'new flatName',
+        street: 'new street',
         city: 'new city',
-        country: 'new country',
         county: 'new county',
+        postalCode: 'new postalCode',
+        country: 'new country',
+        uprn: 'new uprn',
         dependentLocality: 'new dependentLocality',
         doubleDependentLocality: 'new doubleDependentLocality',
-        flatName: 'new flatName',
-        line1: 'new line1',
-        line2: 'new line2',
-        line3: 'new line3',
-        line4: 'new line4',
-        line5: 'new line5',
-        pafOrganisationName: 'new pafOrganisationName',
-        postalCode: 'new postalCode',
-        street: 'new street',
-        uprn: 'new uprn'
+        addressTypeId: undefined
+      },
+      isCorrespondenceAsBusinessAddr: true
+    }
+    const { sbi: _, ...queryReturn } = input
+    queryReturn.address = queryReturn.address.withUprn
+    delete queryReturn.address.withUprn
+    queryReturn.correspondenceAddress = queryReturn.correspondenceAddress.withUprn
+    delete queryReturn.correspondenceAddress.withUprn
```

Comment:
You can do this in one expression:
```suggestion
    const { sbi: _, ...queryReturn } = {
      ...input,
      address: input.address.withUprn,
      correspondenceAddress: input.correspondenceAddress.withUprn
    }
```

Reply:
good point